util/dir_zipper/BUILD.bazel

```diff
@@ -1,4 +1,4 @@
-load("//rust:rust.bzl", "rust_binary")
+load("//rust:defs.bzl", "rust_binary")
```

Comment:
Why these changes?

Reply:
Isn't `//rust:rust.bzl` deprecated/`//rust:defs.bzl` recommended?

Reply:
Yup, but it seems unrelated to this PR. I'm happy to approve a separate PR with this change.

Reply:
Reverted though.